cmd/containerSaveImage.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	piperDocker "github.com/SAP/jenkins-library/pkg/docker"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+
+	"github.com/pkg/errors"
+)
+
+func containerSaveImage(config containerSaveImageOptions, telemetryData *telemetry.CustomData) {
+	var cachePath = "./cache"
+
+	dClientOptions := piperDocker.ClientOptions{ImageName: config.ContainerImage, RegistryURL: config.ContainerRegistryURL, LocalPath: config.FilePath, IncludeLayers: config.IncludeLayers}
+	dClient := &piperDocker.Client{}
+	dClient.SetOptions(dClientOptions)
+
+	err := runContainerSaveImage(&config, telemetryData, cachePath, "", dClient)
+	if err != nil {
+		log.Entry().WithError(err).Fatal("step execution failed")
+	}
+}
+
+func runContainerSaveImage(config *containerSaveImageOptions, telemetryData *telemetry.CustomData, cachePath, rootPath string, dClient piperDocker.Download) error {
+	err := os.RemoveAll(cachePath)
```

Comment:
The naming "cachePath" raises the expectations that image files are cached, but the folder is always emptied. Is this planned for later and therefor reflected in the naming already?

Reply:
no. The tooling requires a path where intermediate files (layers, ...) are stored.
It is done similarly in `protecodeExecuteScan`